src/elements/content-uploader/UploadsManager.scss

```diff
@@ -2,9 +2,14 @@
 
 .be.bcu-uploads-manager-container {
     position: fixed;
+    right: 56px;
     bottom: 0;
-    width: 100%;
+    left: 288px;
```

Comment:
This needs to accomodate the scenario when the left nav is collapsed in a responsive mode. So the actual left positioning of the uploads manager should be $left-nav-width + 32px

$left-nav-width can either be 256px or the smaller/collapsed state (don't know the px off the top of my head)

You'll probably need to import the media queries and variables for left nav collapsed state and set the styles accordingly

Reply:
These styles should live in the parent application, as this component is used by multiple parties, many of whom do not have a left nav at all.